### PR TITLE
Add formula for texmath 0.8.0.2

### PR DESCRIPTION
**Pull request by hollingberry**
This is the first Homebrew formula I've ever created, so any help would be
appreciated!

### CODE REVIEW

File: Library/Formula/texmath.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    cabal_sandbox do
+      cabal_install "--only-dependencies"
+      cabal_install "--prefix=#{prefix}"
+      cabal_install "-fexecutable"
+    end
+    cabal_clean_lib
+  end
+
+  test do
+    system "echo 'a^2 + b^2 = c^2' > pythagorean.tex"
```
Comment (mikemcquaid) [inline Library/Formula/texmath.rb]: Use `(testpath/"pythagorean.tex").write "a^2 + b^2 = c^2"` instead

File: Library/Formula/texmath.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    system "texmath pythagorean.tex > pythagorean.xml"
+    assert (Pathname.pwd/"pythagorean.xml").read.include? "<mn>2</mn>"
```
Comment (mikemcquaid) [inline Library/Formula/texmath.rb]: `assert_match "<mn>2</mn>", shell_output("texmath pythagorean.tex")` rather than the above two lines.

**Comment by mikemcquaid** [comment]
A few warnings that need fixed: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/23546/version=yosemite/testReport/junit/brew-test-bot/yosemite/audit_texmath___strict/

**Comment by hollingberry** [comment]
I think I fixed the problems. Let me know how it looks.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `texmath 0.8.0.2 (new formula)`.

**Pull request closed without merging**